pkg/core/error_handler.go

```
@@ -46,24 +46,23 @@ type FieldEncodingErrorManager func()
 
 func (h *ErrorHandler) EncodeField(field Field) FieldEncodingErrorManager {
 	if h.ShouldSkip() {
-		return func() {
-			// This function is intentionally left empty.
-			// When the ErrorHandler is in EarlyFailing mode and an error has already occurred,
-			// subsequent field encoding operations should be skipped entirely.
-			// Returning a no-op function is the most efficient way to achieve this.
-		}
+		return h.doNothing
 	}
 	maybeFallbackField := h.handleError(field, field.Encode(h.enc))
```

golangci-lint (2.5.0)

[error] missing whitespace above this line (invalid statement above assign)

(wsl_v5)